scripts/binary/build.ts

```diff
@@ -102,7 +102,12 @@ export async function buildCypressApp (options: BuildCypressAppOpts) {
   if (!keepBuild) {
     log('#buildPackages')
 
-    await execa('yarn', ['lerna', 'run', 'build-prod', '--ignore', 'cli', '--concurrency', '4'], {
+    await execa('yarn', ['lerna', 'run', 'build', '--concurrency', '4'], {
```

Comment:
We want to make sure that everything is built here before we build the binary. These will be cached because we already built them in a previous step. We can probably remove this in the future